subprojects/core/src/main/java/org/gradle/internal/vfs/GlobalCache.java

```diff
@@ -20,15 +20,14 @@
 import java.util.List;
 
 /**
- * Represents a store of cached and immutable files.
+ * Represents a location for global Gradle caches.
  *
- * Over time more files can be added to an additive cache, but existing files
- * can never be modified. Files in additive caches can only be deleted as part
- * of cleanup.
+ * The global cache is managed by Gradle, so we Gradle needs to take care
+ * of informing all the infrastructure about changes to it.
  */
-public interface AdditiveCache {
+public interface GlobalCache {
```

Comment:
I don't think these should be in the `org.gradle.internal.vfs` package. We probably shouldn't keep anything in that package in `:core`, but these classes don't strike me as VFS-related at all.

Reply:
Moved to `org.gradle.cache` in `persistentCache`.